source/contracts/reporting/Market.sol

```diff
@@ -324,8 +332,20 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         require(_forkingMarket != this);
         require(!isFinalized());
         IInitialReporter _initialParticipant = getInitialReporter();
+        // Early out if already disavowed or nothing to disavow
+        if (_initialParticipant.getReportTimestamp() == 0) {
```

Comment:
Note, not feedback: If an initial reporter has not been been reported on, you can both keep the original InitialReporter (keeping the designated reporter/using the open-reporting rules) and know that no disputeCrowdsourcers exist.